verification/curator-service/ui/src/components/SourceTable.tsx

```diff
@@ -93,6 +95,8 @@ interface TableRow {
     awsScheduleExpression?: string;
     // dateFilter
     dateFilter?: DateFilter;
+    // notificationRecipients
```

Comment:
Comment doesn't add much value :) remove? or describe in other words what it is

Reply:
Yeah, axed a couple of these.